app/views/pieces/show.html.haml

```diff
@@ -13,16 +13,32 @@
 %div= render 'reviews/form'
 
 - if !@reviews.nil?
-  %ul
+  %ul.clearfix#review_block
     - @reviews.each do |review|
       -if !review.title.empty?
-        %li= review.title
-        %li= review.body
-        %li= "written by: #{review.user.username}"
-        - unless current_user.nil?
-          - if current_user.can_edit?(review)
-            .links
-              %p= link_to "Edit Review", edit_piece_review_path(@piece, review)
-              %p= link_to "Delete Review", piece_review_path(@piece, review), method: :delete
+        .review-container{id: "review_id-#{review.id}"}
```

Comment:
Hash attribute should start with one space after the opening brace<br>Hash attribute should end with one space before the closing brace